interface/VideoEncoderDefs.h

```diff
@@ -155,6 +155,18 @@ typedef struct VideoEncOutputBuffer {
 #endif
 }VideoEncOutputBuffer;
 
+#ifdef __BUILD_GET_MV__
+typedef struct VideoEncMVBuffer {
+    uint8_t *data;
```

Comment:
How many bytes need allocated to data?Where to get it?Do we need add something like IVideoEncoder.getMaxOutSize?